### PR TITLE
Ubuntu Noninteractive

### PR DESCRIPTION
**Pull request by sean-morris**
The build was stopping without this line waiting
 for the user to choose their region

### CODE REVIEW

File: otter/grade/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ARG BASE_IMAGE=ubuntu:22.04
-
+
```
Comment (chrispyles) [inline otter/grade/Dockerfile:2]: Please remove this added whitespace.

**Review by chrispyles — APPROVED** [review]

**Comment by chrispyles** [comment]
Hey @sean-morris - IIRC we were discussing an issue related to this in the otter slack a couple weeks ago -- was that resolved or is there more that needs to be added to this PR? v5.1.0 is going out in a couple of days so I'd like to include this.

**Comment by sean-morris** [comment]
> Hey @sean-morris - IIRC we were discussing an issue related to this in the otter slack a couple weeks ago -- was that resolved or is there more that needs to be added to this PR? v5.1.0 is going out in a couple of days so I'd like to include this.

@chrispyles I do not think anything else is needed. I am on vacation and have not had a chance to get back to experimenting with v5. I think this will work though if I am remembering correctly.

**Comment by chrispyles** [comment]
Thanks for the update @sean-morris. I'll go ahead and merge this.

**Pull request merged**